opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -83,6 +86,13 @@ public long finishMicros() {
         return new ArrayList<>(this.logEntries);
     }
 
+    /**
+     * @return a copy of errors.
+     */
+    public List<RuntimeException> errors() {
```

Comment:
how about `generatedErrors()`? I want to somehow clarify that these are not errors in the `setTag("error", true)` sense.